### DISCUSSION TITLE
Feature Request Last Received via mqtt

### DISCUSSION THREAD

**Discussion by barni** [tbnobody/OpenDTU#280] [Ideas]
Hallo zusammen,

ich fände es super wenn 
"Data Age:" oder der Zeitpunkt des letzen Empfangs auch via mqtt verfügbar wäre.

Grüße

**Comment by tbnobody** [comment]
Hm "Data Age" ist etwas gefährlich. Weil wenn der ESP offline ist bliebe Data Age ja einfach stehen. Ein [UnixTimestamp](https://www.unixtimestamp.com) der den Zeitpunkt der letzten empfangenen Statistikdaten anzeigt würde mir spontan einfallen. Dann hätte man die Zeit als Absolutwert in einem Programmiersprachen unabhängigen Format.

**Comment by tbnobody** [comment]
Habe da in b96f0c4dbec5314f155d98378b4cddbfc91f3650 mal was eingebaut...

**Comment by barni** [comment]
Funktioniert super danke ! Ggf könnte die Wert am Anfang auf 1.1.1970 statt sysdate initialisieren. Wobei ist so auch schon super danke !

**Comment by tbnobody** [comment]
das mit 0 initialisieren ist ganz gut. Habe es in f854cdff0339c88a5e58cd2e7e07beddb3097111 implementiert
